sdk/notificationhubs/notification-hubs/src/index.docs.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export * from "./index.js";
```

Comment:
Creating this file can happen when compiling using package.json exports instead of checking it into source code